src/ripple/app/tests/common_ledger.h

```diff
@@ -26,11 +26,13 @@
 #include <ripple/app/misc/CanonicalTXSet.h>
 #include <ripple/app/transactors/Transactor.h>
 #include <ripple/basics/seconds_clock.h>
+#include <ripple/crypto/KeyType.h>
 #include <ripple/json/json_value.h>
 #include <ripple/protocol/Indexes.h>
 #include <ripple/protocol/RippleAddress.h>
 #include <ripple/protocol/STParsedJSON.h>
 #include <ripple/protocol/TxFlags.h>
+#include <ripple/rpc/impl/KeypairForSignature.h>
```

Comment:
Some suggestions:
* `generateKeypair`
* `keypairForSigning`
* `generateKeypairFromJson` and `generateKeypairFromSeed`

PS. The functions are split out because the second is easier to call from unit tests.